registry.json

```diff
@@ -3600,5 +3600,19 @@
     "telegram": "SmaugPool",
     "operatorName": "",
     "github": "SmaugPool"
+  },
+   {
+    "ticker": "PROOF",
+    "poolId": "pool17s9pcsnhgueer24p4lk53ruglr0r8009yxfrqvwgk5gx6r44cpe",
+    "social": {
+      "website": "https://www.proofstakepool.com",
+      "twitter": "",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "",
```

Comment:
Please can you add a TG handle? This must be the handle of a user and not a group or channel.

Reply:
Telegram: sergejmaehler